helix-term/src/commands.rs

```diff
@@ -3829,6 +3831,30 @@ fn goto_last_line_impl(cx: &mut Context, movement: Movement) {
     doc.set_selection(view.id, selection);
 }
 
+fn goto_column(cx: &mut Context) {
+    goto_column_impl(cx, Movement::Move);
+}
+
+fn extend_to_column(cx: &mut Context) {
+    goto_column_impl(cx, Movement::Extend);
+}
+
+fn goto_column_impl(cx: &mut Context, movement: Movement) {
+    if let Some(count) = cx.count {
```

Comment:
Typically the count is assumed to be 1 if not specified. This can be `let count = cx.count()` instead - that will mirror Vim's behavior of going to the line start if the count is not specified